lib/ReactViews/Map/Navigation/MeasureTool.jsx

```diff
@@ -246,4 +250,4 @@ const MeasureTool = createReactClass({
   }
 });
 
-export default MeasureTool;
+export default withTranslation()(MeasureTool);
```

Comment:
Could you please, add a separate named export (e.g. `const MeasureTool = createReactClass({` to `export const MeasureTool = createReactClass({`),

& then update the import (to test without the HOC wrapper)+ update the tests in `test/ReactViews/MeasureToolSpec.jsx` so that the tests pass again? (You can simply pass an empty function as the `t` prop for now, while we figure out a better way to pull them apart: `const measureTool = <MeasureTool terria={terria} t={() => {}} />;`)

The docs around tests are located here:
https://docs.terria.io/guide/contributing/development-environment/#tests-specs

Reply:
Tests are updated, it is passing all tests at my end.

Reply:
Great!!!